static/js/place/chart_block.tsx

```diff
@@ -266,39 +266,31 @@ class ChartBlock extends React.Component<ChartBlockPropType, unknown> {
       }
       if (
         !!this.props.data.isChoropleth &&
```

Comment:
also add the isUsaPlace check if you still see a flash of content (was removed for the other comparisons)